components/org.wso2.carbon.identity.oauth.dcr.endpoint/src/test/java/org/wso2/carbon/identity/oauth2/dcr/endpoint/TestUtil.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.wso2.carbon.identity.oauth2.dcr.endpoint;
+
+import org.wso2.carbon.base.CarbonBaseConstants;
+import org.wso2.carbon.context.PrivilegedCarbonContext;
+
+public class TestUtil{
+    public static void startTenantFlow (String tenantDomain) {
+        String carbonHome = TestUtil.class.getResource("/").getFile();
```

Comment:
Add a new line after method declaration. Fix in all the places in this PR

Reply:
Fixed - https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1252